tree/30_generic_methods/_classes_copy.cf

```diff
@@ -33,6 +33,7 @@ bundle agent _classes_copy(source_prefix, destination_prefix)
       "prefix" slist => { "promise_kept", "promise_repaired", "repair_failed", "repair_denied", "repair_timeout" };
       "suffix" slist => { "repaired", "ok", "reached", "failed", "not_ok", "not_kept", "not_repaired", "denied", "timeout", "kept", "error" };
 
+    destination_ok::
```

Comment:
In master, this is named destination_exists.
The merge will be easier if you use the same name